src/containers/BookDetails.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import BookShape from '../shapes/book';
+import { fetchBookDetails } from '../redux/actions';
+
+export class BookDetails extends Component {
+  componentDidMount() {
+    const { match } = this.props;
+    this.props.fetchBook(match.params.isbn);
+  }
+
+  render() {
+    const { loading, book, match, error } = this.props;
+    return (
+      <Fragment>
+      <h3>Book Details:</h3>
+      { loading ? <div>Loading book...</div> : null }
+      { error ? <div>Could not load book with isbn {match.params.isbn}</div> : null }
+      { book ?
```

Comment:
I would avoid multiline ternaries, they are hard to read, especially for beginners

Reply:
Fixed it. Thx!